.travis.yml

```diff
@@ -0,0 +1,82 @@
+language: python
+python:
+- '3.4'
+cache: pip
+install:
+- pip3 install https://github.com/jnewland/home-assistant/zipball/rancho-de-newland
+script:
+- openssl req -x509 -newkey rsa:1024 -subj '/CN=example.com' -nodes -keyout /tmp/key.pem
```

Comment:
Kinda wasteful, I could shove a cert into the repo but hey Travis will spend the CPU cycles for me for free so why not I guess?